src/views/datasources/list/DataSourcesListFilters.ts

```diff
@@ -5,12 +5,14 @@ import { RowFilter } from '@openshift-console/dynamic-plugin-sdk';
 
 import { getDataSourceCronJob } from '../utils';
 
-export const getDataImportCronFilter = (t: TFunction): RowFilter<V1beta1DataSource>[] => [
+export const getDataImportCronFilter = (
+  t: TFunction<'plugin__kubevirt-plugin', undefined>,
```

Comment:
Maybe it's worth adding
```ts
export type KubevirtTFunction = TFunction<'plugin__kubevirt-plugin', undefined>;
```
(or omit the second type argument, unless it's strictly required)

to `src/utils/hooks/useKubevirtTranslation.ts` and update the code to use `KubevirtTFunction` consistently throughout the codebase.

Reply:
That sounds great to me! I'd like do this in a followup PR.